examples/uniaxialCompression/uniaxialCompression.m

```diff
@@ -133,5 +187,5 @@
 set(labx, 'FontSize', plotfontsize); set(laby, 'FontSize', plotfontsize) ;
 %md
 %md```@raw html
```

Comment:
I suggest including the plots before merging. You can check that in the deploy https://onsas.github.io/ONSAS.m/previews/PR600/examples/uniaxialCompression/

Reply:
Done